code/game/objects/items/storage/marine_boxes.dm

```diff
@@ -1,47 +1,78 @@
-/obj/item/storage/box/heavy_armor
+/obj/item/storage/box/crate
+	name = "crate"
+	desc = "It's just an ordinary wooden crate."
+	icon = 'icons/obj/items/storage/storage.dmi'
+	icon_state = "case"
+	foldable = /obj/item/stack/sheet/wood
+
+// Copied from C:\Dev\SS13\TerraGov-Marine-Corps\code\game\objects\items\storage\storage.dm
+/obj/item/storage/box/crate/attack_self(mob/user)
+	//Clicking on itself will empty it, if it has the verb to do that.
+	if(allow_quick_empty)
+		quick_empty()
+		return
+
+	//Otherwise we'll try to fold it.
+	if ( length(contents) )
+		return
+
+	if ( !ispath(foldable) )
+		return
+
+	// Close any open UI windows first
+	for(var/mob/M in content_watchers)
+		close(M)
+
+	// Now make the plank
+	to_chat(user, span_notice("You broken [src] into a plank."))
+	new foldable(get_turf(src))
+	qdel(src)
```

Comment:
why does this exist?

Reply:
It's to change the message when it's folded from "You fold [src] flat." to "You broken [src] into a plank."
I'm very new to coding MD so don't know how to do this better. Do I use ..() or something?

Reply:
either make the parent message more generic "you break down the [src]" or something, or add like a break down quote var